metadata.rb

```diff
@@ -8,7 +8,7 @@
 source_url       'https://github.com/sous-chefs/nagios'
 chef_version     '>= 14'
 
-depends 'apache2', '>= 4.0'
+depends 'apache2', '~> 5.0'
```

Comment:
This sort of dep is usually a recipe for disaster in a cookbook. Which version do we actually need for this to work?

Reply:
Cookbook still uses recipes for apache2, not going to rewrite it as part of this.

We should make a list of cookbooks using definitions (this one does) as a hit list

Reply:
So >= 4, <= 6